Game/lv/Game.po

```diff
@@ -2202,182 +2202,182 @@ msgstr "... nav jēgas spriest \"kas būtu, ja\"."
 msgctxt "ST_RadioStation,Dispatcher01a"
 msgid "Unknown (over radio): COMMENCE EXPUNGE, OBSERVER CLEANSE"
-msgstr ""
+msgstr "Nezināms (rācija): SĀKT TĪRĪŠANU, NOVĒROTĀJA TĪRĪŠANA"
 
 #. Key:	Dispatcher01b
 #. SourceLocation:	/Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 #: /Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 msgctxt "ST_RadioStation,Dispatcher01b"
 msgid "72 DEGREES, 59 MINUTES AND 6 SECONDS NORTH, 2 DEGREES, 51 MINUTES AND 54 SECONDS EAST"
-msgstr ""
+msgstr "72 GRĀDI, 59 MINŪTES UN 6 SEKUNDES ZIEMEĻU VIRZIENĀ, 51 MINŪTES UN 54 SEKUNDES AUSTRUMU VIRZIENĀ"
 
 #. Key:	John01a
 #. SourceLocation:	/Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 #: /Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 msgctxt "ST_RadioStation,John01a"
 msgid "John Derril (over radio): Wait what's that... is that the radio? Hello? Hello! Oh God yes you did it!"
-msgstr ""
+msgstr "Džons Derils (rācija): Paga, kas tad tas...vai tā ir rācija? Sveiki? Halo! Ak Kungs tu to paveici!" 
 
 #. Key:	John01b
 #. SourceLocation:	/Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 #: /Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 msgctxt "ST_RadioStation,John01b"
 msgid "Is there anyone there?! This is an SOS, I repeat, an SOS"
-msgstr ""
+msgstr "Vai te kāds ir ?! Šis ir SOS, es atkārtoju SOS"
 
 #. Key:	John02a
 #. SourceLocation:	/Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 #: /Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 msgctxt "ST_RadioStation,John02a"
 msgid "Does anyone copy? The island has been overrun."
-msgstr ""
+msgstr "Vai kāds kopē? Šī sala ir pārpildīta."
 
 #. Key:	John02b
 #. SourceLocation:	/Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 #: /Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 msgctxt "ST_RadioStation,John02b"
 msgid "Arque Corporation have been stopping ALL broadcast messages from the island. We need IMMEDIATE millitary asssistance!"
-msgstr ""
+msgstr "Aruq Korporācija ir apturējuis VISUS ziņojumus no salas. Mums ir nepieciešama TŪLĪTĒJA palīdzība!"
 
 #. Key:	John03a
 #. SourceLocation:	/Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 #: /Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 msgctxt "ST_RadioStation,John03a"
 msgid "John Derril (over radio): We're at approximately 72 degrees, 59 minutes and 6 seconds North, and uh, 2 degrees, 51 minutes and 54 seconds East."
-msgstr ""
+msgstr "Džons Derils (rācija): Esam apmēram 72 grādus, 59 minūtes un 6 sekundes uz ziemeļiem, un ā, 2 grādus, 51 minūti un 54 sekundes uz austrumiem."
 
 #. Key:	John04a
 #. SourceLocation:	/Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 #: /Game/Resources/Localisation/ST_RadioStation.ST_RadioStation
 msgctxt "ST_RadioStation,John04a"
 msgid "John Derril (over radio): Uh, it's John."
-msgstr ""
+msgstr "Džons Derils (rācija) Ē, esmu Džons."
```

Comment:
Would you mind keeping the same formatting? `Džons Derils (rācija): Ē, esmu Džons.`

Reply:
Fixed it!

By the way, can we discuss about the Estranged: Act I elevator issue? I reported it on Discord (uploaded a video) and even by your personal email but you didn't respond.

Reply:
thanks! let's talk on Discord - I am actually not looking to update Estranged: Act I (to change content would mean braking saved games)